openqa-investigate

```diff
@@ -374,7 +374,22 @@ investigate() {
     comment_id=$(sync_via_investigation_comment "$id" "$first_cluster_job_id") || rc=$?
     [[ $rc != 255 ]] && return $rc
 
-    out=$(trigger_jobs "$id" "${@:2}") || return $?
+    out=""
+    if ! out=$(trigger_jobs "$id" "${@:2}"); then
+        first_comment_id=$(client-get-job-comments "$first_cluster_job_id" | runjq -r '[.[] | select(.text | contains("Investigation aborted"))] | min_by(.id) | .id')
+        if [[ $comment_id != "$first_comment_id" ]]; then
+            client-delete-job-comment "$first_cluster_job_id" "$comment_id"
```

Comment:
I don't really get the logic here.
If `trigger_jobs` fails, you are searching for comments that contain "Investigation aborted", which only happen if `trigger_jobs` failed, so I'm wondering why we would find existing comments like this.
And then you delete this comment, but a few lines later you try to update this comment with `PUT`, but it might have been deleted.

Reply:
I've added this after the test I've made in my local environment if the investigation is runned multiple time the abort comment will be published over and over again. With the requested change you made in https://github.com/os-autoinst/scripts/pull/398#discussion_r2075173772 the problem is solved because that comment trigger line 188 and stop the script with "Skipping investigation of job 1: job cluster is already being investigated, see comment on job 1"

If is the case is correct, I'll remove this change!